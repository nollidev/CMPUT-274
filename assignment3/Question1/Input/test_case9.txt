31 4
W BP BC - BJ BG BN BH - - BM BD BL - - - - AB AA BB BE - BA BF BK - BO AC AD BI X
