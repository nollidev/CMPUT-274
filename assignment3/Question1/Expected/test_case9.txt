W -> BP BC BJ
BP -> BG BN BH
BG -> BA BF BK
BN -> BO AC AD
BH -> BI X
BC -> BM BD BL
BJ -> AB AA BB BE
